agimus_controller/theos_work/main.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
```

Comment:
This files should be name `agimus_controller/main_hpp_mpc.py`
Don't keep it in `theos_work`.